README.md

```diff
@@ -63,14 +63,14 @@ end
 **Loaf** adds `breadcrumb` helper also to the views. Together with controller breadcrumbs, the view breadcrumbs are appended as the last. For instance, to specify view breadcrumb do:
 
 ```ruby
-<% breadcrumb "#{@category.title}", blog_category_path(@category) %>
+<%= breadcrumb @category.title, blog_category_path(@category) %>
```

Comment:
In this case, I don't think it's correct to use `=`, because we are adding a new breadcrumb to the list not printing them.